### DISCUSSION TITLE
Dynamic Optimization in Casadi with .xml file

### DISCUSSION THREAD

**Discussion by mario35to** [OpenModelica/OpenModelica#11917] [Q&A]
Hello all,

I exported my model in OMEdit as .xml file to use Casadi in OMPython. I followed the Chapter "Dynamic Optimization with OpenModelica and CasADi" in the https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/optimization.html . But couldn't perform 3. step "Run defaultStart.py python script from OpenModelica optimization directory".
I saw another post about the same problem in Openmodelica forum: https://openmodelica.org/forum/default-topic/2809-optimization-with-openmodelica-import-the-model-in-casadi.html .
Is this method become out-of-date? 
If so, is pymoca updated and usable method? I found an example at pymoca github https://github.com/pymoca/pymoca/blob/master/test/notebooks/XML.ipynb 
Thanks in advance.

**Comment by casella** [comment]
@arun3688 can you please help @mario35to?
